### PR TITLE
Seperate services to be their own module

### PR DESCRIPTION
**Pull request by chang47**
I used Spring Initializer to move each project to be their own module instead of having everything inside of one mega project. This better adheres to the idea of microservices.

This also has the added benefit of solving the problem where we were not able to access the database because of the Eureka services. I have confirmed that we can now successfully add entries into our local DB

Another extra benefit is that we now also have a test folder!

### CODE REVIEW

**Comment by chang47** [comment]
I don't understand the WebServer code, it seems like its 2 projects? So I decided to leave it as is.

File: Atomizer/pom.xml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+			<artifactId>spring-boot-starter-web</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-starter-netflix-eureka-client</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-starter-netflix-eureka-server</artifactId>
+		</dependency>
```
Comment (senghuot) [inline Atomizer/pom.xml]: reminds me again why we need to include this since Atomizer is just a client

Comment (chang47) [reply Atomizer/pom.xml]: I was just checking to see if you actually read my PR's.

JK. Good point, I was just following the tutorial and it also included it. It did strike me as odd.

Comment (chang47) [reply Atomizer/pom.xml]: Looks like it's not needed and there's a bug in the tutorial. Updating.

**Review by senghuot — COMMENTED** [review]
Another thing is that we prolly need to make Kakuna a folder to separate out each service. But we can think about this in the future commit.

File: Atomizer/pom.xml (hunk 1 above)
Comment (senghuot) [reply Atomizer/pom.xml]: So you're admitting i was right you hoe lol #FeelsGood #Blessed

**Comment by chang47** [comment]
Interested in understadng what you mean. I re-purposed the Kakuna folder to be the root. Each module is its own service, no?

File: Atomizer/pom.xml (hunk 1 above)
Comment (chang47) [reply Atomizer/pom.xml]: You review your co-workers with that mouth?

Comment (senghuot) [reply Atomizer/pom.xml]: okay i take it back my bad. i forgot this is public.

**Comment by senghuot** [comment]
> Interested in understadng what you mean. I re-purposed the Kakuna folder to be the root. Each module is its own service, no?

Say if you want to deploy say Atomizer, currently, we have to pull down the whole Kakuna repo down then compile the whole thing. IMO, this is not a good practice, on top of that ideally each service code base should be standalone. So my proposal is to make Kakuna like collection of repo or some sort for the service. We will think about this more as we go along

**Comment by chang47** [comment]
@senghuot Ah yeah that makes sense, but since this whole thing is just an example I think it makes sense just to keep it together for convenience.

**Pull request merged**